src/app/Plans/components/PlanActionsDropdown.tsx

```diff
@@ -11,6 +25,7 @@ const PlansActionsDropdown: React.FunctionComponent = () => {
       isPlain
       dropdownItems={[
         <DropdownItem
+          isDisabled={hasNotCondition(PlanStatusConditionsType.Ready)}
```

Comment:
Are we sure that it will lose the ready condition once it's in-progress? If it's errored, does it consider itself ready for a retry? not sure how this data looks

Reply:
According to @jortel the conditions has one entry "Ready" if it's ready otherwise it shouldn't be present.

Reply:
Good question, @jortel could help us out here please?

Reply:
@mturley , sorry, I was wrong. There will be an explicit condition "Execute".